recipe/meta.yaml

```diff
@@ -1,31 +1,45 @@
 {% set version = "3.1.4" %}
+{% set commit = "71f5b2d5b68192424fce6bd1768a889710fc5933" %}
 
 package:
   name: mpfr
   version: {{ version }}
 
 source:
-  fn: mpfr-{{ version }}.tar.bz2
-  url: http://ftp.gnu.org/gnu/mpfr/mpfr-{{ version }}.tar.gz
-  md5: 482ab3c120ffc959f631b4ba9ec59a46
+  fn: mpfr-{{ version }}.tar.bz2                                            # [unix]
+  url: http://ftp.gnu.org/gnu/mpfr/mpfr-{{ version }}.tar.gz                # [unix]
+  md5: 482ab3c120ffc959f631b4ba9ec59a46                                     # [unix]
+  fn: {{ commit }}.zip                                                      # [win]
+  url: https://github.com/BrianGladman/mpfr/archive/{{ commit }}.zip        # [win]
+  md5: 9d84b0d1ac32244e837d1b220b0ceb73                                     # [win]
 
 build:
   number: 1
-  skip: true  # [win]
+  skip: true          # [win and not py35]
+  features:
+    - vc14            # [win and py35]
```

Comment:
Do we still need `python` in `requirements/build`, @msarahan and @pelson? While this looks ok now, I'm concerned that a re-rendering PR will come along and remove `CONDA_PY` from the AppVeyor file, which could cause us some issues.